src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -6,8 +6,7 @@
                     <label for="git-branch" class="col-sm-2 col-form-label text-right">Git branch</label>
                     <div class="col-sm-6">
                         <select class="form-control" id="git-branch" v-model="selectedBranch" @change="changedBranch">
-                            <option disabled selected value=""> -- Select a branch -- </option>
```

Comment:
I also just removed this and set to the first branch by default - does that make sense? Just seemed like 1 less click for the user if it defaults to master.

Reply:
Yeah, that seems fine. Can't remember how the branches are ordered, does it make sure that master is always first?

Reply:
No idea tbh, it comes back from orderly.server. We could do an explicit search for "master" (although of course people might be using "main" or something else.)

Reply:
Let's leave it as default for now then. I wonder if we might need to add in a search at some point, could end up with a lt of branches..

Reply:
Yeah v true